src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -2016,6 +2021,7 @@ private String init(boolean initFull) {
         
         configureTools = externalToolService.findByType(ExternalTool.Type.CONFIGURE);
         exploreTools = externalToolService.findByType(ExternalTool.Type.EXPLORE);
```

Comment:
findByType isn't filtering by scope yet - should these use findByTypeAndScope with file scope?

Reply:
Yes, they probably should.